package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "type": "commonjs",
   "scripts": {
+    "prepare": "husky install",
```

Comment:
Why husky doesn't autoinstall anymore: https://blog.typicode.com/husky-git-hooks-autoinstall

Reply:
It looks like we just need `postinstall`: https://typicode.github.io/husky/#/?id=yarn-2